tests/stakingRewards/StakingRewards.js

```diff
@@ -44,6 +44,7 @@ const {
 } = require("../Utils/initializer.js");
 
 const StakingLogic = artifacts.require("StakingMock");
+const StakingLogicMockup = artifacts.require("StakingMockup");
```

Comment:
looks like it is better to move added weight by block mocking functionality to StakingMock

Reply:
decided to postpone for refactoring